src/api/python/CMakeLists.txt

```diff
@@ -136,7 +138,7 @@ if ("${IN_VIRTUALENV}" STREQUAL "NO")
   set(INSTALL_CMD "${INSTALL_CMD}
           --prefix=${CMAKE_INSTALL_PREFIX}
           --single-version-externally-managed
-          --record=pycvc5-installed-files.txt")
+          --record=${BASE_PY_API_PACKAGE_NAME}-installed-files.txt")
```

Comment:
Is this ever used?

Reply:
I'm not sure why it's important to make this file.